server/wizard/steps/step_create_zoom_app.go

```diff
@@ -0,0 +1,33 @@
+package steps
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/flow/steps"
+)
+
+const (
+	stepNameCreateApp = "create_app"
+
+	stepTitleCreateApp = ""
+
+	stepDescriptionCreateApp = `1. Enter a name for your app, such as "Mattermost Plugin".
+2. Choose **Account-level app** as the app type.
```

Comment:
Shouldn't we have a step for Account level app and another step for User Level app?

Reply:
@larkox This effort is focused on getting set up quickly, so we are going with one method to keep the setup process simple. Linking to the documentation at the beginning and end of the flow will allow the admin to discover other options for set up.